### DISCUSSION TITLE
is there any function like sequelize.literal in prisma?

### DISCUSSION THREAD

**Discussion by rizqonmaulana** [prisma/prisma#16415] [Q&A] [answered]
I need to combine the ORM function to get data with some SQL raw query, I don't want to write the whole raw query, is there any function like sequelize.literal in prisma?

I want to create a query like bellow

`SELECT type, uid, 
  CONVERT(
    Aes_decrypt(
      phone, 
      Sha2('SECRET', 512), 
      'KEY'
    ) USING utf8
  ) AS phone
FROM 
  Accounts`

In Sequelize I usually write something like bellow

`Accounts.findAll({
    attributes: {
        'type',
        'uid',
        [sequelize.literal('CONVERT(AES_DECRYPT(phone, SHA2('SECRET',512), 'KEY') USING utf8)'), 'phone']
    }
});`

**Comment by nurul3101** [comment, marked as answer]
Hello @rizqonmaulana 👋 

An alternative to Sequalize literal doesn't exist yet, but we do have a feature request of `whereRaw` which intend to achieve similar functionality: #5560, if you can leave a 👍 to the feature request then it would help us in prioritising it.

**Reply by rizqonmaulana** [reply]
Ok got it, thanks for your reply, I think need to write the whole raw query for now

**Reply by nurul3101** [reply]
That's correct, you would need to use a raw query for now.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
